### PR TITLE
Add Process > Build an Outline

### PR DESCRIPTION
**Pull request by joshuacc**
It's been a while, but here's a new section for Prose for Programmers. Would appreciate review if anyone has time. :pencil:

### CODE REVIEW

File: manuscript/process.md [hunk 1]
```diff
@@ -237,4 +237,74 @@ Revising the outline to incorporate this narrative gives us:
     * Refer to parable
     * Supporting factual details
 
-Combining these three structures creates a much more powerful and persuasive account any of them alone.
\ No newline at end of file
+Combining these three structures creates a much more powerful and persuasive account any of them alone.
```
Comment (ghost) [inline manuscript/process.md]: _than_ any of them?

File: manuscript/process.md [hunk 2]
```diff
@@ -237,4 +237,74 @@ Revising the outline to incorporate this narrative gives us:
...
+* Point 2
+* Point 3
+* Point 4
+* Point 5
+
+Breadth-first generally produces better results,
+because it provides more overall context each time we step down a level of detail.
+That extra context makes it much less likely that we will get stuck on trivia.
+
+### Evaulate and revise
```
Comment (ghost) [inline manuscript/process.md]: Evaluate

**Comment by ghost** [comment]
Looks good to me -- happy to see a new section!

**Comment by joshuacc** [comment]
@lgierth Thanks for looking it over!

**Pull request merged**